02c_layout.ipynb

```diff
@@ -81,10 +89,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "id": "3d909af2",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "NameError",
+     "evalue": "name 'selected_data_grid' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[6], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;66;03m# Display dataframe\u001b[39;00m\n\u001b[0;32m----> 2\u001b[0m \u001b[43mselected_data_grid\u001b[49m\n",
+      "\u001b[0;31mNameError\u001b[0m: name 'selected_data_grid' is not defined"
```

Comment:
This is a real error, will note the source of the error in 02b

Reply:
Where is this error?  Are you sure it is not because you don't have the assumed correct module files in the `dashboard` directory?  Copy the `key/dashboard` to `dashboard` and see if the result changes.  I am not seeing this error.

Reply:
Not sure what happened, but when I made sure to start with no `dashboard` directory when I started, it worked. Some of what went wrong, I think, is that some of the early exercises required adding nbdev directives that I might have missed.